src/osa/scripts/closer.py

```diff
@@ -157,6 +158,12 @@ def post_process(seq_tuple):
         list_job_id = merge_dl1_datacheck(seq_list)
         daily_datacheck(daily_longterm_cmd(list_job_id))
```

Comment:
so daily_datacheck function should return the job_id then pass it to `cherenkov_transparency` which should call sbatch as 
```python
        "sbatch",
        "-D",
        options.directory,
        "-o",
        "log/cherenkov_transparency_%j.log",
        f"--dependency=afterok:{longterm_job_id)}",
        "lstchain_cherenkov_transparency",
        "blabla"
```